_posts/2015-10-12-how-to-install.md

```diff
@@ -78,7 +78,7 @@ It seems that sometimes the virtualbox kernel modules are not working correctly
 ### MacOS X
 
 1. [Install Xcode](https://developer.apple.com/xcode/downloads/): `xcode-select --install`
-2. [Install Vagrant](http://docs.vagrantup.com/v2/installation/) (version [2.0.4](https://releases.hashicorp.com/vagrant/2.0.4/), or any other release before 2.1.0!)
```

Comment:
Why remove this link?

Reply:
That was a mistake on my part. I just fixed it.